java/fury-core/src/main/java/org/apache/fury/util/unsafe/_JDKAccess.java

```diff
@@ -65,7 +65,7 @@ public class _JDKAccess {
       property = property.substring(2);
     }
     String jmvName = System.getProperty("java.vm.name", "");
```

Comment:
Nit: `jmvName` should be renamed to `jvmName`.

Reply:
Nice catch, fixed in https://github.com/apache/incubator-fury/pull/1439